src/components/QueryBuilder/index.js

```diff
@@ -0,0 +1,175 @@
+import { useEffect, useState } from "react";
+import { TrashIcon } from "@heroicons/react/solid";
```

Comment:
We can use react-icons here? react-icon seems to have most icons. Long term, we can try removing heroicons.
UTY